components/measurement/SummaryText.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import moment from 'moment'
+import { Text } from 'ooni-components'
+import { getTestMetadata } from '../utils'
+
+const SummaryText = ({
+  testName,
+  network,
+  country,
+  date,
+  hint = 'is reachable',
+  testUrl,
+
+}) => {
+  const metadata = getTestMetadata(testName)
+  const formattedDate = moment(date).format('LL')
+  const formattedDateTime = moment(date).format('lll')
+  return (
+    <Text py={4} fontSize={3}>
+      The <a href={metadata.info}>{metadata.name}</a> test
+      of {testUrl && <a href={testUrl}>{testUrl}</a>} in the {network} network
```

Comment:
We should be sure to not make these links to the tested website. This can lead the SEO of our website being badly affected and we don't want people submitting data to OONI to get indexed and crawled.